### DISCUSSION TITLE
Launch a new chat

### DISCUSSION THREAD

**Discussion by Carpea03** [Beyond-Better/bb#63] [Ideas]
<img width="2240" alt="Screenshot 2024-12-18 at 8 52 36 am" src="https://github.com/user-attachments/assets/654a6131-4476-431c-bdff-d5b4d952dd36" />

From this page there should be a way to launch a new chat within the selected project

**Comment by cngarrison** [comment]
Thanks for the feedback. That's already on the list and will be coming to a release soon.

**Comment by cngarrison** [comment]
This has been added in v0.5.11.
